dolphinscheduler-ui/.env.development

```diff
@@ -15,4 +15,4 @@
 
 NODE_ENV=development
 
-VITE_APP_DEV_WEB_URL='http://127.0.0.1:12345'
+VITE_APP_DEV_WEB_URL='http://8.142.34.29:12345'
```

Comment:
It's better to set to the default ip.

Reply:
Addressed in https://github.com/apache/dolphinscheduler/pull/11226/commits/7782c5d6cf93a4473fcdf8288ee433ed61870388.